storybook/stories/configProvider/displayAliases.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+import * as React from 'react'
+import { ConfigProvider, Grid } from 'gymnast'
+import { colors } from '../../shared'
+
+export default () => {
+  const displayAliases = {
+    test: {
+      minWidth: '351px',
+      maxWidth: '600px',
+    },
+    test2: [
+      {
+        maxWidth: '350px',
+      },
+      {
+        minWidth: '601px',
+      },
+    ],
+  }
+
+  return (
+    <ConfigProvider displayAliases={displayAliases}>
+      <Grid justify="center">
+        <Grid show="test" style={colors.colors1} margin="L" size={6}>
```

Comment:
What do you think about renaming `show` to `showWhen`?

Reply:
If we want to make changes to the resolution api now is the time (I think only PES is using it)

Since this is part of the existing API, if we change it I wouldn't make it as part of this PR but we can update it